server/models/procedures.sql

```diff
@@ -639,18 +639,23 @@ BEGIN
       /*
         A positive remainder means that the debtor overpaid slightly and we should debit
         the difference to the debtor and credit the difference as a gain to the gain_account
-
-        - The debtor entity an invoice reference are not included on the gain
-          account transaction. In this case the debtor covered MORE than the 
-          invoiced amount and so referencing them on the enterprise gain will 
-          actually debit them the additional amount. 
       */
       IF (remainder > 0) THEN
-  
-        -- The debtor is not debited in this transaction. They have already 
-        -- balanced the invoice and their debt according to the invoice (the 
-        -- exact amount). The additional payment can just be put in a gain account.
-        
+
+        -- debit the debtor
+        INSERT INTO posting_journal (
```

Comment:
I think this is wrong.  However, I'll let @sfount review it.

@sfount, can  you make sure this posting work is correct?